### PR TITLE
cleanup and remove old mc settings

### PR DESCRIPTION
**Pull request by miltontony**

### CODE REVIEW

File: mc2/settings.py [hunk 1]
```diff
@@ -178,7 +177,6 @@ def abspath(*args):
 ROOT_URLCONF = 'mc2.urls'
 
 # Python dotted path to the WSGI application used by Django's server.
-# WSGI_APPLICATION = 'ws4redis.django_runserver.application'
```
Comment (jerith) [inline mc2/settings.py]: Do we still need the first line of this comment?

**Comment by jerith** [comment]
The test failure looks relevant.

**Comment by miltontony** [comment]
@jerith I've ratified the changes.

**Comment by miltontony** [comment]
@jerith @calston ready for review

**Comment by jerith** [comment]
:+1:

**Pull request merged**